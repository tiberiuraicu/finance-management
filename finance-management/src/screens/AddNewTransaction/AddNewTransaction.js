import React from "react";
import { View } from "react-native";

const AddNewTransaction = () => {

  return (<View></View>);
};

export default AddNewTransaction;