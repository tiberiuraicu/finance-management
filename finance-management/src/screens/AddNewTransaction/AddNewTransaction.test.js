import AddNewTransaction from "./AddNewTransaction.js";
import { shallow } from 'enzyme';

describe('AddNewTransaction screen tests', () => {
    it('renders the component', () => {
      const wrapper = shallow(<AddNewTransaction />);
      expect(wrapper).toMatchSnapshot();
    });
  });