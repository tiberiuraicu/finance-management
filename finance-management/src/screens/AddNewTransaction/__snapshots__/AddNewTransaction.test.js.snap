// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddNewTransaction screen tests renders the component 1`] = `ShallowWrapper {}`;
